frame/support/src/lib.rs

```diff
@@ -84,12 +84,13 @@ pub use sp_runtime::{self, ConsensusEngineId, print, traits::Printable};
 #[derive(Debug)]
 pub enum Never {}
 
-/// Macro for easily creating a new implementation of the `Get` trait. Use similarly to
-/// how you would declare a `const`:
+/// Macro for easily creating a new implementation of the `Get` trait. If `const` token is used, the
```

Comment:
Please improve the docs and please remove `no_compile` below. We should make sure it compiles.

(I know that you not added the example, but we should improve when we touch it)